src/main/scala/com/fortysevendeg/translatebubble/ui/components/ContentView.scala

```diff
@@ -74,24 +68,23 @@ class ContentView(context: Context, attrs: AttributeSet, defStyleAttr: Int)(impl
     animator.addListener(new Animator.AnimatorListener {
       def onAnimationStart(animation: Animator) {
       }
+
       def onAnimationEnd(animation: Animator) {
         hide()
         params.alpha = 1
         windowManager.updateViewLayout(ContentView.this, params)
       }
+
       def onAnimationCancel(animation: Animator) {
       }
+
       def onAnimationRepeat(animation: Animator) {
       }
     })
     animator.start()
   }
 
-  def changePositionIfIsNecessary(
-      widthScreen: Int,
-      heightScreen: Int,
-      params: WindowManager.LayoutParams,
-      windowManager: WindowManager): Unit = {
+  def changePositionIfIsNecessary(widthScreen: Int, heightScreen: Int, params: WindowManager.LayoutParams, windowManager: WindowManager): Unit = {
```

Comment:
Not sure if this format is better, this line is too long. Maybe you don't have our formatting preferences?